configs/default/IFRC-IFLIGHT_F745_AIO.config

```diff
@@ -123,11 +138,15 @@ led 3 9,11::CTOBVIW:0
 # master
 set baro_bustype = I2C
 set baro_i2c_device = 1
-set baro_hardware = BMP280
+set baro_hardware = AUTO
```

Comment:
Please remove the default settings from the configuration.

Reply:
Fixed.